### ISSUE TITLE
Return source for fatal errors

### ISSUE THREAD

**Issue by IanVS** [eslint/eslint#3325]
Currently, when a parsing (`fatal`) error occurs, the `source` is not included in the message that ESLint returns.  It may be worthwhile to investigate whether the source could be returned, which is used in some formatters to show clearly where the problem is.

**Comment by gyandeeps** [comment]
Do we need to do this here or inside `espree` project.

**Comment by ilyavolodin** [comment]
This should go to espree I think.

**Comment by IanVS** [comment]
Moved to https://github.com/eslint/espree/issues/185

**Comment by nzakas** [comment]
This is an ESLint issue, not an Espree one. ESLint attaches source code to errors.

**Comment by gyandeeps** [comment]
Currently I see this when I run eslint on a invalid javascript code.

```sh
C:\Users\gs025879\Documents\webstrom\test\gyandeep.js
  3:18  error  Parsing error: Unexpected token )

✖ 1 problem (1 error, 0 warnings)
```

Is this what is expected or are we looking for something more than this?

**Comment by IanVS** [comment]
This issue is about returning the source along with the message, so that formatters like [eslint-friendly-formatteer](https://github.com/royriojas/eslint-friendly-formatter#example-of-the-output) can show the code where the error occurred.

**Comment by nzakas** [comment]
@gyandeeps every other error has a `source` property returned. This issue is to add that for fatal errors.
